_posts/2024-04-14-blog.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: post
+title: "Missing data and PCA"
+subtitle: "How little is too little?"
+authors:
+ - Thiseas C. Lamnidis
+categories: Blog
+tag: Blog
+---
+
+_by [Thiseas C. Lamnidis](https://www.eva.mpg.de/archaeogenetics/staff/thiseas-christos-lamnidis/)_
+
+## Introduction
+
+In the field of ancient human population genetics, we often analyse ancient genomes with high proportions of missing data, namely in the recovered genotypes.
+This can stem from limitations of material preservation, and/or of available laboratory/sequencing technologies.
+Given that sampling remains for ancient DNA is a destructive process, we have an ethical responsibility to make the most of the data we are able to generate.
+At the same time, we need to know exactly the limitations of our data when inferring results, and present our findings fairly and compassionately.
+After all, the past breeds the present, and the interpretations we make of the past can have implications affecting present-day peoples and communities.
+
+To that end, archaeogenticists often use quality cutoff thresholds in our analyses, limiting the set of analysed individuals to ones with sufficient SNP coverage for population genetics analyses.
+These cutoffs are often expressed in terms of the "Number of covered SNPs", i.e. the number of SNPs which do not have a missing genotype call for a given ancient individual.
+But what constitutes a sensible minimum cutoff?
+
+Principal Component Analysis (PCA) is a very useful _visualisation_ tool for human aDNA data, as well as the source of many-a [popgen bingo](https://ancientpopgen-bingo.firebaseapp.com/?game=ancientpopgen-bingo).
+It is often the case that most (if not all) sampled individuals will be featured in a PCA plot, and it is important to understand how missing data affects their position in order to avoid misinterpreting PCA results.
```

Comment:
```suggestion
It is often the case that most (if not all) individuals in an ancient population genetics study will be featured in a PCA plot, and it is important to understand how missing data affects their position in order to avoid misinterpreting PCA results.
```